package.json

```diff
@@ -17,8 +17,11 @@
     "testqa2ChromeClinician": "nightwatch --env qa2chrome --tag clinician",
     "testqa2ChromeNotifications": "nightwatch --env qa2chrome --tag notifications",
     "testqa2ChromeRpm": "nightwatch --env qa2chrome --tag rpm2",
-    "testprdChromeNotifications": "nightwatch --env prd --tag notifications",
+    "testprdChromeRpm": "nightwatch --env prdchrome --tag rpm2",
+    "testprdChromeNotifications": "nightwatch --env prdchrome --tag notifications",
     "testqa2ChromeTag": "nightwatch --env qa2chrome --tag clinician",
+    "testCustodial": "nightwatch --env prdchrome,qa2chrome --tag qae222",
```

Comment:
**question:** Do we need this here? I would say, that better place would be *README.md*, that you are able to specify tag somehow and use that as `npx nighwatch --tag qae222`. If we have 20 tests, will we have 20 commands here?

Reply:
it's probably okay to keep this here. it's more of a shortcut for debugging.